backend/main.go

```diff
@@ -77,6 +78,7 @@ func main() {
 	}
 	log.Infof("| consul-address       : %-50s |", cfg.ConsulAddress)
 	log.Infof("| consul.acl-token     : %-50s |", cfg.ConsulACLToken)
+	log.Infof("| consul-top-bar-color : %-50s |", cfg.ConsulColor)
```

Comment:
i think this should match the CLI option, so either rename the CLI  flag to `consul-top-bar-color` or this log statement (and nomad) to `consul-color` 

I think `*-color` make sense, since the color also change the primary color around the app, and not only the top bar :)

Reply:
absolutely right, I actually named them differently and then renamed them. But forgot this one!